### PR TITLE
Add webp decoder option to handle the background color in ANIM chunk

### PR DESCRIPTION
**Pull request by brianpopow**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description
Adds a flag to decide how to handle the background color Animation Chunk.
The specification is vague on how to handle the background color of the animation chunk.
This option let's the user choose how to deal with it.

Related Issue: #2528

### CODE REVIEW

File: src/ImageSharp/Formats/Webp/BackgroundColorHandling.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public enum BackgroundColorHandling
+{
+    /// <summary>
+    /// The background color of the ANIM chunk will be used to initialize the canvas to fill the unused space on the canvas around the frame.
+    /// Also, if AnimationDisposalMethod.Dispose is used, this color will be used to restore the canvas background.
+    /// </summary>
+    Standard = 0,
+
+    /// <summary>
+    /// The background color of the ANIM chunk is ignored and instead the canvas is initialized with black, BGRA(0, 0, 0, 0).
```
Comment (JimBobSquarePants) [inline src/ImageSharp/Formats/Webp/BackgroundColorHandling.cs:18]:
```suggestion
    /// The background color of the ANIM chunk is ignored and instead the canvas is initialized with transparent, BGRA(0, 0, 0, 0).
```

**Review by JimBobSquarePants — COMMENTED** [review]
Looking good so far, hopefully we get permission to use the image soon.

File: tests/ImageSharp.Tests/TestImages.cs [hunk 2]
```diff
@@ -677,6 +677,7 @@ public static class Lossy
             public const string WithXmp = "Webp/xmp_lossy.webp";
             public const string BikeSmall = "Webp/bike_lossy_small.webp";
             public const string Animated = "Webp/leo_animated_lossy.webp";
+            public const string AnimatedIssue2528 = "Webp/issues/issue2528.webp";
```
Comment (antonfirsov) [inline tests/ImageSharp.Tests/TestImages.cs:680]: Maybe fa4febf7fe3264a440c4d4e135dd027882e24fb0 has to be reverted now?

Comment (brianpopow) [reply tests/ImageSharp.Tests/TestImages.cs]: yeah I think you are right. We should squash commits when merging this, because of the back and forth here

**Review by JimBobSquarePants — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**